src/cachetools/ttl.py

```diff
@@ -54,29 +55,31 @@ def __getattr__(self, name):
 class TTLCache(Cache):
     """LRU Cache implementation with per-item time-to-live (TTL) value."""
 
-    def __init__(self, maxsize, ttl, timer=time.monotonic, getsizeof=None):
+    def __init__(self, maxsize, ttl, timer=time.perf_counter, exclusive=False, getsizeof=None):
```

Comment:
I think you will always want to add new parameters to the end of the function definition.

While pytho allows to specify keyword only arguments - that's not done here, so this change would break instances where all parmeters (including `getsizeof`) are given as positional argument - but in the worst possible way - without telling users that the implementation changed by failing completely.

Reply:
`getsizeof` is not present in module `cachetools.func`, and it’s probably not commonly used by users. Put the new argument after `getsizeof` will cause inconsistent argument order between `cachetools.ttl.TTLCache` and `cachetools.func.ttl_cache`.

Reply:
It's a breaking change nonetheless, and one that might lead to bad user/upgrade experience, as it'll break silently (solely based on the assumption that not many people will use this).
Problem is,  if they do, it'll break potentially without them noticing for quite some time - assuming people will not setup tests to test every single parameter of a library they use.

If you think the order MUST be this way, then then i'd suggest to enforce keyword arguments - either for the last 3 (proposed below), but at least for the last 2 arguments (exclusive and getsizeof).

```suggestion
    def __init__(self, maxsize, ttl, *, timer=time.perf_counter, exclusive=False, getsizeof=None):
```

Any project with decent coverage / mypy CI setup will capture this as "breaking" change and therefore alert people to modify their usage of this class.